ui/tests/test-helper.js

```diff
@@ -3,7 +3,11 @@ import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
 import './helpers/flash-message';
+import preloadAssets from 'ember-asset-loader/test-support/preload-assets';
+import manifest from 'vault/config/asset-manifest';
 
-setApplication(Application.create(config.APP));
+preloadAssets(manifest).then(() => {
+  setApplication(Application.create(config.APP));
```

Comment:
Just to be sure, will this load the dependencies for all additional new engines as well?

Reply:
Yeah, any engines with `lazyLoading: true` configured. Here's the docs for it: https://github.com/ember-engines/ember-asset-loader#pre-loading-assets-during-testing